cmd/libs/go2idl/deepcopy-gen/generators/deepcopy.go

```diff
@@ -153,6 +150,10 @@ func (g *genDeepCopy) copyableWithinPackage(t *types.Type) bool {
 	if !strings.HasPrefix(t.Name.Package, "k8s.io/kubernetes/") {
 		return false
 	}
+	// FIXME: Remove it
+	// Becuase of some reason we don't have comment lines for Cloner.
+	// WTF?
+	glog.Errorf("-> %#v %#v", t, t.CommentLines)
```

Comment:
I think that it parses only files being explicitly passed as input dirs - the transitive closure of dependencies is not parsed like that...

heh - I have to come up with something smarter....